CHANGELOG.md

```diff
@@ -3,6 +3,8 @@
 ## Unreleased
 
 - Recreate REST client after starting a workspace to ensure fresh TLS certificates.
+- Use `coder ssh` subcommand in place of `coder vscodessh`.
+- Simplify written SSH config to avoid excessive api calls when configuring the coder cli.
```

Comment:
This was actually only an issue with the JetBrains plugin AFAIK.

Reply:
Ah right, removing, I shouldn't write docs after long meetings 💀